packages/gatsby/src/redux/index.ts

```diff
@@ -129,18 +129,22 @@ export const saveState = (): void => {
   })
 }
 
-export const saveStateForWorkers = (slices: Array<GatsbyStateKeys>): void => {
+export const savePartialStateToDisk = (
+  slices: Array<GatsbyStateKeys>,
+  optionalPrefix: string = ``
```

Comment:
Every worker should save its own stuff so they can pass in their `process.env.GATSBY_WORKER_ID` to prefix their files

Reply:
I am not sure if this prefix should be optional. Is there a scenario when workers (or main process) will save to the same filename?

Reply:
It's optional in the sense that for the main -> worker flow there is no need to prefix it as the main process only saves one file and I didn't want to require an additional arg in those cases